packages/feedback-spa/src/app/feedback/feedback-item/feedback-item.component.ts

```diff
@@ -11,7 +11,7 @@ export class FeedbackItemComponent implements OnInit {
   @Input() jiraURL: any;
   @Input() roverURL: any;
   @Output() clickAction: EventEmitter<any> = new EventEmitter<any>();
-  isAccordionOpen: Boolean | any = false;
+  isAccordionOpen: boolean | any = false;
```

Comment:
Never use any as an alternative to some other type. Also, try to avoid using any.

Reply:
i am going to use like this *isAccordionOpen = false;*

Reply:
if i will remove any it will through me another warning **"Type boolean trivially inferred from a boolean literal, remove type annotation (no-inferrable-types)"**

Reply:
That means that you do not need to define the type as `boolean` since it is already being inferred from the initial value.
So just keep it
```ts
isAccordionOpen = false;
```